tests/jballerina-unit-test/src/test/resources/test-src/workers/basic-worker-actions.bal

```diff
@@ -119,6 +120,241 @@ function workerSameThreadTest() returns any {
     return res;
 }
 
+function simpleSendActionWithCloneableType() returns string {
+    worker w1Cloneable returns boolean {
+        value:Cloneable w1a = 10;
+        value:Cloneable w1b = 11;
+        w1a -> w2Cloneable;
+        w1b ->> w2Cloneable;
+        return true;
+    }
+
+    worker w2Cloneable returns boolean {
+        value:Cloneable w2a = 15;
+        value:Cloneable w2b = 15;
+        sleep(10);
+        w2a = <- w1Cloneable;
+        w2b = <- w1Cloneable;
+        if (w2a is int) {
+            assertValueEquality(10, w2a);
+        } else {
+            return false;
+        }
+        if (w2b is int) {
+            assertValueEquality(11, w2b);
+        } else {
+            return false;
+        }
+        return true;
+    }
+
+    record { boolean w1Cloneable; boolean w2Cloneable; } x = wait {w1Cloneable, w2Cloneable};
+    assertValueEquality(true, x.w1Cloneable);
+    assertValueEquality(true, x.w2Cloneable);
+    return "done";
```

Comment:
Since the call to `simpleSendActionWithCloneableType` is blocking and we wait for the workers here, IMO we don't have to return a value here and assert it. WDYT?

Reply:
I think we can also change the rest of the tests similarly.

Reply:
Fixed https://github.com/ballerina-platform/ballerina-lang/pull/30250/commits/1ee63a7c47b0cd174504e4502ed20acc07f005cc